src/AppBundle/Resources/translations/odr-assets.en.yml

```diff
@@ -106,6 +106,7 @@ form:
       label: What is %client%’s share of the property?
     isSubjectToEquityRelease:
       label: Is this property subject to an equity release scheme?
+      hint: This means that %client%'s stake, or equity, in the property has been used to generate a lump sum or income
```

Comment:
I've gone through all the changes and fixed missing hint text and all the %client% which wasn't showing.

I'm merging this now.